surefire-providers/pom.xml

```diff
@@ -41,6 +41,7 @@
     <module>surefire-junit3</module>
     <module>surefire-junit4</module>
     <module>surefire-junit47</module>
+    <module>surefire-junit5</module>
```

Comment:
Agreed. Please do not refer to the _JUnit Platform_ as "JUnit 5".

The JUnit Platform is approaching version 1.0 GA, not 5.0.